### DISCUSSION TITLE
Why WebKit instead of Servo?

### DISCUSSION THREAD

**Discussion by ghost** [OkuBrowser/oku#322] [Q&A]
https://github.com/servo/servo

**Comment by emmyoh** [comment]
Hello,

I mention this briefly on the [About page](https://okubrowser.github.io/about/), but to get more into detail: Servo is not mature enough to meet the needs of the project. I'd be open to starting an experimental branch that uses Servo instead of WebKit, but I'd need to be able to embed Servo as a GTK component and AFAIK that's not yet possible.

Will post updates in the future if the status on this ever changes.
